### ISSUE TITLE
Homework not bound to a class

### ISSUE THREAD

**Issue by amazansky** [MyHomeworkSpace/client#128]
Users should have the option to make homework/other assignments that aren't bound to a specific class. If this is too hard to do, then an easy alternative might be to make a class called "Other" for new users and automatically assign it to any HW entries that are made without a class.

**Comment by thatoddmailbox** [comment]
The problem with that is I'm not really sure where that would go in the UI. Like on the planner, we could add a "no class" row, but that seems like effectively the same thing as an "Other" class. 

If we do just autocreate an Other class, then we'd need some way to select it as the default. I guess we could also just look for a class titled Other and make that the default?

**Comment by amazansky** [comment]
Yeah, if we do end up choosing to make a no class row we could treat it as a class in the backend. Otherwise just an "other" class would be fine.
